### ISSUE TITLE
A11y_Visual Studio Code Client_Welcome Screen_Screen reader: Narrator/NVDA is not announcing the correct label name "show welcome page on startup".

### ISSUE THREAD

**Issue by Shivanikupatkar** [microsoft/vscode#137210]
### GitHubTags
#A11yTCS; #A11ySev2; #A11yMAS; #A11yWCAG; #DesktopApp; #Visual Studio Code Client;#BM-VisualStudioCode-Win32-Nov2021; #WCAG1.3.1; #Win32; #Element:label;#ScreenReader; #NVDA; #Narrator;#Win10;#AINotInScope;

### Actual Result
Narrator/NVDA is not announcing the correct label name of " show welcome page on startup".

### Expected Result
Narrator/NVDA should announce the correct label name of "show welcome page on startup".

### Environment Details
Application Name: Visual Studio Code Client
Version:1.62.1 (user setup)
Windows Version: Windows 10 (21H1)

### Repro Steps
1)Start Narrator/NVDA.
2)Open Visual studio code editor
3)Using "Alt+H" key navigate to the "Help" menu bar option .
4)Using "Down" arrow key navigate to the "Welcome" item.
5) Make sure you have the following  setting in settings.json: "workbench.startupEditor": "gettingStarted"
6)Using "Alt+H" navigate to "Help" menu bar option and use "down" arrow key to navigate to "Getting started" and observe that the screen reader is announcing the correct label name of the "show welcome page on startup". 

### User Impact
Screen reader users will face difficulty as correct label name is not announce by the screen reader.

**Comment by isidorn** [comment]
This is expected behaviour. Closing as invalid.
Screen reader always announces the name of the menu entry as it should. The "Get Started" label is appropriate here
